docs/guides/software-lifecycle/continuous-testing/README.md

```diff
@@ -1,3 +1,716 @@
+- **Security Tests:** Identifies potential security vulnerabilities.
+- **Build Tests:** Checks if the code builds into binaries or packages successfully.
+- **Acceptance Tests:** Validates against end-user and stakeholder requirements.
+
+Once you've identified the testing categories important to your project, write a description for each using the provided template section. Be sure to include:
+
+1. The purpose of that testing category.
+2. The location where the tests are stored.
+3. How the tests are triggered.
+4. The framework used for testing.
+5. Best practices or guidelines for contributing to that testing category.
+
+#### 1.2 Example: Unit Testing
+
+In the "Unit Testing" section, explain how unit tests are structured, where they're stored, and how frequently they run. Include the testing framework used and best practices for writing unit tests. For example:
+
+- **Location:** `/tests/unit`
+- **Purpose:** Verify that individual functions or components work as intended.
+- **Running Tests:**
+  - **Manually:**
+    1. Navigate to the `/tests/unit` directory.
+    2. Run the relevant unit testing command (e.g., `npm test` for JavaScript).
+    3. Review results in the terminal or generated reports.
+  - **Automatically:**
+    - **Frequency:** Upon every commit or pull request.
+    - **Results Location:** [GitHub Actions Unit Tests Workflow](#)
+- **Framework Used:** Jest for JavaScript.
+- **Tips:** Focus on core functions and methods, use mocks for dependencies, and handle edge cases.
+
+#### 1.3 Example: Performance Testing
+
+In the "Performance Testing" section, detail how you ensure your application can handle expected and peak loads. For example:
+
+- **Location:** `/tests/performance`
+- **Purpose:** Validate that the application meets performance goals under load.
+- **Running Tests:**
+  - **Manually:**
+    1. Navigate to the `/tests/performance` directory.
+    2. Run the performance testing tool command (e.g., `jmeter -n -t test.jmx`).
+    3. Review generated reports for analysis.
+  - **Automatically:**
+    - **Frequency:** Monthly stress tests or before release candidates.
+    - **Results Location:** deployment test server's `/var/log/myapp/performance-tests.log
+- **Framework Used:** Apache JMeter.
+- **Tips:** Simulate peak load scenarios, monitor resource usage, and use tools like Chaos Monkey for resilience testing.
+
+### 2. Write Your Tests
+
+We recommend using [Testing Frameworks](testing-frameworks) when writing your tests to automate, organize, and analyze your testing efforts effectively. You can write your test code from scratch, but another way to write it is using large language models (LLMs). 
+
+We recommend adding inline comments in your tests to clarify the purpose of each test. These comments should include details on the function being tested, the test type (e.g., bug fix, change request, requirements validation, anomaly reports), and any relevant context. For example: 
```

Comment:
The inline comments block is too long and placed in a bit of a confusing spot. You're talking about LLMs prior to that block and immediately after, so it might make sense to put that block elsewhere and shorten it.